src/components/common/MediaCard.js

```diff
@@ -64,10 +66,7 @@ const MediaCard = props => {
 
       <Collapse disableSpacing in={expanded} timeout="auto" unmountOnExit>
         <CardContent className={classes.cardDescription}>
-          <Typography
-            component="p"
-            dangerouslySetInnerHTML={{ __html: text }}
-          />
+          <Text content={text} className="MediaCardDescriptionText" />
```

Comment:
If following the steps above, here you could add `className={classes.cardDescriptionText}` instead of relying on the `css`.